src/themes/masschallenge/globals/site.variables

```diff
@@ -16,6 +16,7 @@
 @textColor: #676767;
 @greyBackground: #efebeb;
 @whiteColor: #fff;
+@borderWidth: 1px;
```

Comment:
I'm not sure if `@borderWidth` should be a constant variable. If it is, perhaps it should at least be more descriptive, for example `@borderWidthThin` so that we can also have `@borderWidthMed: 2px;` or `@borderWidthThick: 3px`.